src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetChangelogPage.tsx

```diff
@@ -28,11 +29,14 @@ export default function ReleaseApiDataSetChangelogPage() {
     data: dataSet,
     isLoading: isLoadingDataSet,
     refetch: refetchDataSet,
+    isError: errorFetchingDataSet,
   } = useQuery(apiDataSetQueries.get(dataSetId));
 
-  const { data: changes, isLoading: isLoadingChanges } = useQuery(
-    apiDataSetVersionQueries.getChanges(dataSetVersionId),
-  );
+  const {
+    data: changes,
+    isLoading: isLoadingChanges,
+    isError: errorFetchingChanges,
```

Comment:
As above - we can infer this when `changes` is undefined and loading has completed. In the error state, `changes` is not defined.

Reply:
Done ta